modules/heat_conduction/src/fvbcs/FVFunctorConvectiveHeatFluxBC.C

```diff
@@ -0,0 +1,49 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "FVFunctorConvectiveHeatFluxBC.h"
+#include "Function.h"
+
+registerMooseObject("HeatConductionApp", FVFunctorConvectiveHeatFluxBC);
+
+InputParameters
+FVFunctorConvectiveHeatFluxBC::validParams()
+{
+  InputParameters params = FVFluxBC::validParams();
+  params.addClassDescription(
+      "Convective heat transfer boundary condition with temperature and heat "
+      "transfer coefficent given by functors.");
+  params.addRequiredParam<MooseFunctorName>("T_wall", "Functor for wall temperature");
```

Comment:
so wall is solid?
and infinity is fluid?

infinity is usually bulk btw

Reply:
Yes, renamed in the next commit to T_solid and T_bulk